.github/workflows/check-release.yaml

```diff
@@ -0,0 +1,18 @@
+on: push
+
+name: R-CMD-check
```

Comment:
This should be at the top of the file and should be maybe have a better name.  At least one that isn't the same as the action below.